### DISCUSSION TITLE
Pull requests for modifications which I'm not planning to upstream

### DISCUSSION THREAD

**Discussion by Steffo99** [cheeaun/phanpy#777] [Q&A] [answered]
I've made a few personalizations to my personal instance of Phanpy, mostly related to the stylesheet.

I'm not interested in upstreaming them, since they're stylistic choices that many people might not agree with; however, if you want, I can open draft pull requests for them where I describe what I changed, so that they're easier to find if someone wants to get a couple features from them.

Would you prefer me to open those or not?

**Comment by Steffo99** [comment]
Wait maybe I should ping @cheeaun...

**Comment by cheeaun** [comment, marked as answer]
Sorry for the late response (almost forgot about this).

I'm fine with PRs, especially if you could like split them nicely as in one feature per PR (I know this is tough) so that it's easier to review and merge. I'm also fine with you listing out what are the modifications you've done as they could serve as ideas for my experiments, where I could probably come up with something better (hopefully).

**Reply by Steffo99** [reply]
perfect!

i'll cherry-pick them soon(ish), then!

(right now i'm spending all my coding time for university things, so i don't have much energy left to upstream things in the short term...)
